library/ctr_drbg.c

```diff
@@ -354,15 +402,22 @@ int mbedtls_ctr_drbg_random_with_add( void *p_rng,
         ctx->prediction_resistance )
     {
         if( ( ret = mbedtls_ctr_drbg_reseed( ctx, additional, add_len ) ) != 0 )
+        {
             return( ret );
-
+	}
```

Comment:
Style: Please change to spaces instead of tabs

Reply:
OK